tests/testsuite/run.rs

```diff
@@ -22,30 +22,34 @@ fn simple() {
 }
 
 #[cargo_test]
-fn simple_quiet() {
+fn quiet_arg() {
```

Comment:
The tests in this file are specifically testing the behavior of `cargo run`, so I don't think they can be changed to `cargo build`.  I would suggest leaving them as `cargo run`. I would also suggest sticking with `with_stderr` and `with_stdout` just to validate the full output.

Reply:
Ah yeah, of course. I'll update, thanks.